packages/starlight/components/SidebarSublist.astro

```diff
@@ -43,6 +47,30 @@ interface Props {
     margin-top: 0.75rem;
   }
 
```

Comment:
Ah, and remembering we also need these styles to get rid of the default arrow in Safari:
```suggestion

  summary::marker,
  summary::-webkit-details-marker {
    display: none;
  }
```
Looks like this currently 😅 
<img width="282" alt="image" src="https://github.com/withastro/starlight/assets/357379/4e294dc1-2779-4f21-b79f-d5c854f7678f">

Reply:
The dreaded Safari